Description: Do not detect filenames starting with end,even,odd as keywords
Origin: upstream, version 1.43
Forwarded: not-needed

--- a/pdftk/pdftk.cc
+++ b/pdftk/pdftk.cc
@@ -1073,6 +1073,15 @@
     int keyword_len= 0;
     keyword arg_keyword= is_keyword( argv[ii], &keyword_len );
 
+		// these keywords can be false hits because of their loose matching requirements;
+		// since they are suffixes to page ranges, their appearance here is most likely a false match;
+		if( arg_keyword== end_k ||
+				arg_keyword== even_k ||
+				arg_keyword== odd_k )
+			{
+				arg_keyword= none_k;
+			}
+
     switch( arg_state ) {
 
     case input_files_e: 
